operator/cmd/mesh/install.go

```diff
@@ -74,12 +77,12 @@ type installArgs struct {
 
 func addInstallFlags(cmd *cobra.Command, args *installArgs) {
 	cmd.PersistentFlags().StringSliceVarP(&args.inFilenames, "filename", "f", nil, filenameFlagHelpStr)
-	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", KubeConfigFlagHelpStr)
-	cmd.PersistentFlags().StringVar(&args.context, "context", "", ContextFlagHelpStr)
+	args.kubeConfigFlags.AddFlags(cmd.PersistentFlags())
```

Comment:
seems inconsistent to use clioptions for just these two. recommend sticking to separate for consistency with codebase.

Reply:
Done. Reverted.